scripts/api-compatibility-check/breakage_allowlist.txt

```diff
@@ -0,0 +1,7 @@
+// https://github.com/mapbox/mapbox-maps-ios/pull/1095
+Func MapView.didMoveToSuperview() has been removed
+Func MapView.willMove(toWindow:) has been removed
+
+// _spi(Experimental)
+Enum PinchGestureBehavior has been removed
+Var GestureOptions.pinchBehavior has been removed
```

Comment:
@macdrevx I'd like to name each breaking change in this file (and remove them just after the new public release). What is the best name for these functions, WDYT?